app/src/main/res/values/colors.xml

```diff
@@ -6,5 +6,12 @@
     <color name="teal_200">#FF03DAC5</color>
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
+
+
     <color name="white">#FFFFFFFF</color>
+    <color name="landing_button_blue">#23A7F1</color>
```

Comment:
I would suggest to rename colors as they are, for example just blue or blue_500 etc. Ideally, in a project you should not have more than 10 - 20 colors pallete and screens or views should not have different colors.